stdlib/public/core/FloatingPointParsing.swift.gyb

```diff
@@ -120,16 +148,21 @@ extension ${Self}: LosslessStringConvertible {
   ///       // p?.isNaN == true
   ///       // String(p!) == "nan(0x10)"
   ///
-  /// Passing any other format or any additional characters as `text` results
-  /// in `nil`. For example, the following conversions result in `nil`:
+  /// - A string in any other format or containing additional characters
+  ///   results in a `nil` value. For example, the following conversions
+  ///   result in `nil`:
+  ///
+  ///       ${Self}(" 5.0")      // Includes whitespace
+  ///       ${Self}("±2.0")      // Invalid character
+  ///       ${Self}("0x1.25e4")  // Incorrect exponent format
+  ///
+  /// - Parameter text: An input string to convert to a `${Self}?` instance.
   ///
-  ///     ${Self}(" 5.0")      // Includes whitespace
-  ///     ${Self}("±2.0")      // Invalid character
-  ///     ${Self}("0x1.25e4")  // Incorrect exponent format
+  /// Backwards compatibility note:  Prior to Swift 5.4, a decimal or
+  /// hexadecimal input string whose value was too large to represent
+  /// as a finite `${Self}` instance would return `nil` instead of
+  /// `${Self}.infinity`.
```

Comment:
How about putting this in an aside:

```suggestion
  /// - Note: Prior to Swift 5.4, a decimal or
  /// hexadecimal input string whose value was too large to represent
  /// as a finite `${Self}` instance returned `nil` instead of
  /// `${Self}.infinity`.
```

Reply:
I moved the rest of the rounding discussion down to the end to consolidate.